spyder/plugins/workingdirectory/container.py

```diff
@@ -337,7 +349,8 @@ def get_workdir(self):
     @Slot(str)
     @Slot(str, bool)
     @Slot(str, bool, bool)
-    def chdir(self, directory, browsing_history=False, emit=True):
+    @Slot(str, bool, bool, str)
+    def chdir(self, directory, browsing_history=False, emit=True, server_id=None):
```

Comment:
```suggestion
    def chdir(
        self, directory, browsing_history=False, emit=True, server_id=None
    ):
```

- For 79 columns.
- Also, please document the new parameter in the docsstring of this method.